ublox-short-range/src/command/edm/mod.rs

```diff
@@ -75,15 +76,7 @@ where
             at_resp = &resp[AT_COMMAND_POSITION..pos];
         }
 
-        self.0.parse(at_resp)
-    }
-
-    fn force_receive_state(&self) -> bool {
-        true
-    }
-
-    fn max_timeout_ms(&self) -> u32 {
-        self.0.max_timeout_ms()
+        self.0.parse(Ok(at_resp))
```

Comment:
This might not work as intended, as the idea is that `parse()` should take the error if `resp` is `Err()`. 
I think the correct way would be to `match resp` rather than `resp?` on the top line.